### DISCUSSION TITLE
a/make-windows11-install-u-disk-on-macos

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#11] [Announcements]
# a/make-windows11-install-u-disk-on-macos

https://blog.yasking.org/a/make-windows11-install-u-disk-on-macos.html

**Comment by EasonAI-5589** [comment]
太强了哥们，感谢你的教程和分享

**Reply by sincerefly** [reply]
哈哈，感谢回复，很高兴这篇笔记对你有所帮助 🌹

**Comment by SiyuWang15** [comment]
我直接cp过去，还是有一个install.wim文件超过4G 拷贝不了。。。。我只能改成exFat格式化了，客服说那块主板bios支持exFat，等装机完了我试试。。。

**Reply by sincerefly** [reply]
试了成功没  👀

**Reply by SiyuWang15** [reply]
成功了，但是其实我没有用系统盘启动，硬盘里本来就有windows系统，直接setup.exe运行就行了。

**Comment by AndyLuo1029** [comment]
成功了，感谢大佬！

**Reply by sincerefly** [reply]
哈哈，感谢回复 ❤️
